elastio-fargate-mysql-backup/README.md

```diff
@@ -0,0 +1,49 @@
+# AWS RDS MySQL backup by Elastio and AWS Fargate
+
+## Create AIM Role for Elastio Backup
+
+1. Go to [AWS IAM Roles](https://console.aws.amazon.com/iamv2/home#/roles)
+2. Press Create Role button
+3. Select Elastic Container Service Task under Elastic Container Service for AWS service trusted entity and press Next
+![image](https://github.com/elastio/contrib/assets/81738703/0a7050a0-895b-4227-a609-40bb9c6acb24)
+4. Select `ElastioBackupAdmin` permission and press Next
+5. Type ElastioMySQLBackupRole as a role name and press Create role
```

Comment:
Again, please use `fixed width` fonts every place you specify a role name, variable name, CLI command, etc

```suggestion
5. Type `ElastioMySQLBackupRole` as a role name and press Create role
```